lib/ru_propisju.rb

```diff
@@ -325,6 +380,18 @@ module RuPropisju
       :eur_fraction => ["євроцент", "євроцента", "євроцентів"],
       :usd_integral => ["долар", "долара", "доларів"],
       :usd_fraction => ['цент', 'цента', 'центів'],
+      :chf_integral => ['швейцарський франк', 'швейцарських франків', 'швейцарських франків'],
```

Comment:
Not quite right, I'll leave the correct option below
```suggestion
      :chf_integral => ['швейцарський франк', 'швейцарських франки', 'швейцарських франків'],
```